.github/workflows/convertimages.yml

```
@@ -0,0 +1,141 @@
+name: Convert Images
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+permissions:
+  actions: read
+  checks: read
+  contents: write
+  deployments: none
+  issues: read
+  packages: none
+  pull-requests: write
+  repository-projects: none
+  security-events: none
+  statuses: none
+
+jobs:
+  build:
+    runs-on: 'ubuntu-22.04'
+    name: Convert Images
+    steps:
+      - name: Detect branch for PR
+        id: vars
+        run: |
+          PR="${{ github.event.pull_request.number }}"
+
+          PR_INFO=$( curl \
+                   --request GET \
+                   --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+                   --header 'content-type: application/json' \
+                   --url https://api.github.com/repos/$GITHUB_REPOSITORY/pulls/$PR )
+          REF=$(echo "${PR_INFO}" | jq -r .head.ref)
+          BASE=$(echo "${PR_INFO}" | jq -r .head.repo.full_name)
+          STATE=$(echo "${PR_INFO}" | jq -r .state)
+          BASE_SHA=$(echo "${PR_INFO}" | jq -r .base.sha)
+
+          if [[ $STATE == "closed" ]]
+          then
+            echo "Pull Request already closed."
+            exit 0;
+          fi
+
+          echo "islocalbranch=$(expr $BASE == $GITHUB_REPOSITORY)" >> $GITHUB_OUTPUT
+          echo "branch=$REF" >> $GITHUB_OUTPUT
+          echo "base=$BASE_SHA" >> $GITHUB_OUTPUT
+      - uses: actions/checkout@v4
+        if: steps.vars.outputs.branch != ''
+      - name: Check images changes
+        id: imagecheck
+        run: |
+          git fetch --depth=1 origin ${{ steps.vars.outputs.base }}
+
+          IMAGES_MODIFIED=$(git diff --name-only ${{ steps.vars.outputs.base }} -- src/ .github/workflows/convertimages.yml | wc -l)
+
+          if [[ $IMAGES_MODIFIED == "0" ]]
+          then
+            echo "No images modified"
+            exit 0;
+          fi
+
+          echo "images_modified=1" >> $GITHUB_OUTPUT
+        if: steps.vars.outputs.branch != ''
+      - name: Install Dependencies
+        run: |
+          npm install
+
+          sudo apt-get update
+
+          git clone --recursive https://github.com/kornelski/pngquant.git
+          cd pngquant
+          cargo build --release
+          sudo cp target/release/pngquant /usr/local/bin/
+          cd ..
+          pngquant --version
+
+          sudo add-apt-repository universe
+          sudo add-apt-repository ppa:inkscape.dev/stable
+          sudo apt-get update
+          sudo apt install inkscape
+
+          sudo apt install imagemagick

```

```suggestion
          sudo apt-get install inkscape

          sudo apt-get install imagemagick
```

Just to get rid of those "apt does not have a stable CLI interface" warnings in the log.